pandas/tests/io/test_html.py

```diff
@@ -65,9 +65,6 @@ def _skip_if_none_of(module_names):
                 pytest.skip("Bad version of bs4: 4.2.0")
 
 
-DATA_PATH = tm.get_data_path()
```

Comment:
This types of changes were because `pytest.skip` can't be called outside of test methods.